app/utils/publication-filter.js

```diff
@@ -44,6 +44,15 @@ class PublicationFilter {
       withdrawnFilterOption: this.withdrawnFilterOption,
     };
   }
+
+  hasFilter() {
```

Comment:
Sorry, nitpicking. Ik zou er hier een get-function van maken. Dan wordt die getter in de controller overbodig.

Reply:
Aha daarom wilde die niet correct werken :D